### DISCUSSION TITLE
New interface

### DISCUSSION THREAD

**Discussion by Rogala** [Comfy-Org/ComfyUI#5318] [Q&A] [answered]
How to enable system boot indication?
![image](https://github.com/user-attachments/assets/f827ffae-287b-4dd5-91e2-8af77b1a4c0d)

**Comment by LukeG89** [comment, marked as answer]
Resources Monitor comes from [ComfyUI-Crystools](https://github.com/crystian/ComfyUI-Crystools)

**Reply by Rogala** [reply]
Thank you very much.
